app/database/group_store.go

```diff
@@ -21,6 +21,7 @@ func (s *GroupStore) TeamGroupByTeamItemAndUser(itemID int64, user *User) *DB {
 				groups_groups.idGroupChild = ?`, user.SelfGroupID).
 		Where("groups.idTeamItem = ?", itemID).
 		Where("groups.sType = 'Team'").
+		Order("groups.ID").
```

Comment:
This default ordering should be added in the function spec so that we know which one is returned in case there are several ones.

Reply:
fixed